src/evergreen/api.py

```diff
@@ -1076,7 +1076,7 @@ def get_users_for_role(self, role: str) -> List[str]:
         :param role: Role to fetch users for.
         """
         url = self._create_url(f"/roles/{role}/users")
-        return self._call_api(url, method="GET").json()
+        return self._call_api(url, method="GET").json()["users"]
```

Comment:
Could we create a model class for the json returned? We do that for all the other json returned from the API and it makes it easier to understand what data is being returned.

Reply:
Given the response is - {"users": ["user_id1", "user_id2", "user_id3" .... ]} - it seems like we can either have a `UsersForRole` model or return the raw response as the code here suggests. 

I think that a `UsersForRole` model class is a bit unusual. I considered having just a `User` model class but the issue is that https://github.com/evergreen-ci/evergreen.py/blob/main/src/evergreen/base.py#L66 expects a dict.

I'll defer to what you think is the best approach :)

Reply:
I would lean towards having a `UsersForRole` class, since that is the pattern that the other endpoints use. It also provides a path forward if the endpoint starts to return more complex results in the future.

I'm not sure what you mean about the class being unusual. Since the json being returned is a dict, it would still take a dict. I would think it would look something like this:
```python
class UsersForRole(_BaseEvergreenObject):
    users = evg_attrib("users")

    ...
```